packages/zowe-explorer/__tests__/__unit__/job/actions.unit.test.ts

```diff
@@ -339,33 +342,35 @@ describe("Jobs Actions Unit Tests - Function downloadSingleSpool", () => {
             path: "",
             query: "",
         };
-        jobs.push(node);
+        jobs.push(node, node);
```

Comment:
Is there a reason why the node was added twice into the jobs array? Since this is logic in tests, I don't have any request for changes here; was mostly curious 😋

Reply:
We need at least 2 nodes to be able to test multiple branches - 1 where the node contains spool files and 1 where the node doesn't:
`jest.spyOn(SpoolProvider, "getSpoolFiles").mockResolvedValueOnce([spool]).mockResolvedValueOnce([]);`